students/M_Sunday/lesson03/list_lab.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+
+# lesson 03 - list lab exercise
+
+
+def list_contents(list_, id_=1):
```

Comment:
It doesn't look like you're ever passing in a different value for the `id_` parameter, so it can be removed from this function.